### PR TITLE
ci: cloudfront deployments

### PR DESCRIPTION
**Pull request by hejkerooo**

### CODE REVIEW

File: .github/workflows/cf-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Clear bucket
+        run: |
+          aws s3 rm s3://${{ secrets.BUCKET_NAME }} --recursive
+      - name: Upload to bucket
+        run: |
+          aws s3 cp dist s3://${{ secrets.BUCKET_NAME }} --recursive
+      - name: Invalidate CloudFront
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml:75]: Why this `Invalidate Cloudfront` step?

**Review by jrhender — COMMENTED** [review]
Thanks for the PR @hejkerooo . Are you going to be taking on https://energyweb.atlassian.net/browse/DOI-1439 or is this just a PR to share your code?

File: .github/workflows/cf-deploy.yml (hunk 1 above)
Comment (hejkerooo) [reply .github/workflows/cf-deploy.yml:75]: Yes, I'm taking care of it.

Cloudfront caches s3 content so after each deployment, it needs to be cleared.

Comment (jrhender) [reply .github/workflows/cf-deploy.yml:77]: Thanks, what do you think of adding this as a comment?
```suggestion
      # cloudfront caches s3 content so after each deployment, it needs to be cleared.
      - name: Invalidate CloudFront
```

File: .github/workflows/cf-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        run: |
+          aws cloudfront create-invalidation --distribution-id ${{ secrets.CLOUDFRONT_ID }} --paths "/*"
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml:79]: Will each environment of SB have its own `distribution-id`/`CLOUDFRONT_ID`, so a deploy to one environment won't invalidate the cache of the other environments?

**Review by jrhender — COMMENTED** [review]
Just a couple more comments

File: .github/workflows/cf-deploy.yml (hunk 2 above)
Comment (hejkerooo) [reply .github/workflows/cf-deploy.yml:79]: Yes

**Review by dawidgil — APPROVED** [review]

**Pull request merged**